src/dev/flang/tools/docs/Html.java

```diff
@@ -138,6 +138,34 @@ private String summary(AbstractFeature af)
   }
 
 
+  /**
+   * list of features that are redefined by feature af
+   * @param af
+   * @return list of redefined features, as HTML
+   */
+  private String redefines(AbstractFeature af)
+  {
+    if (af.redefines().isEmpty())
+      {
+        return "";
```

Comment:
try to avoid having multiple `return` statements, makes the code harder to understand and debug.